src/pocketmine/command/SimpleCommandMap.php

```diff
@@ -176,6 +176,25 @@ public function register(string $fallbackPrefix, Command $command, string $label
 
 	/**
 	 * @param Command $command
+	 *
+	 * @return bool
+	 */
+	public function unregister(Command $command) : bool{
+		$label = $command->getLabel();
+
+		foreach($this->knownCommands as $lbl => $cmd) {
```

Comment:
you don't need to check the label here, just check if `$cmd` is equal to `$command`.